composer.json

```diff
@@ -15,9 +15,7 @@
 			"role": "Developer"
 		}
 	],
-	"license": [
-		"GPL-2.0+"
-	],
+	"license": "GPL-2.0",
```

Comment:
Should stay "GPL-2.0+" to indicate that any later version is fine too.

https://spdx.org/spdx-specification-21-web-version#h.twlc0ztnng3b